py/specs/count.toml

```diff
@@ -1,5 +1,5 @@
 [setup]
```

Comment:
I guess it the intention was to move this file into the select/ folder, then this file should be removed